<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterInterface</class>
 <widget class="QWidget" name="MasterInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QFrame" name="MasterFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>576</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QGroupBox" name="SideGroupBox">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>128</y>
      <width>110</width>
      <height>310</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QFrame" name="SideFrame">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>5</y>
       <width>95</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QRadioButton" name="HomePageRadioButton">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>20</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="MatchHistoryPageRadioButton">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>120</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="OtherFunctionPageRadioButton">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>220</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="SideLabel">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>115</y>
       <width>20</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ClosePushButton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>4</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AppLogoLabel">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>128</y>
      <width>90</width>
      <height>266</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../ResourceFiles/Resources.qrc">:/ResourceFiles/BasicFiles/AppLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="EchoTextLabel">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>398</y>
      <width>600</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="EchoerTextLabel">
    <property name="geometry">
     <rect>
      <x>647</x>
      <y>438</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../ResourceFiles/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
